src/main/java/Main.java

```diff
@@ -0,0 +1,55 @@
+import figure.AbstractFigure;
+import figure.Circle;
+import figure.Square;
+import figure.Trapezoid;
+import figure.Triangle;
+
+import static figure.Colors.getRandomColors;
+
+public class Main {
+    public static void main(String[] args) {
+        final int FIGURES_COUNT = 4 + (int) (Math.random() * 10);
```

Comment:
let's replace it with an ordinary variable
constant fields should be declared at class level